.github/workflows/ci.yaml

```diff
@@ -79,14 +79,19 @@ jobs:
         uses: JafarAbdi/latest-rosdistro-release-date-action@main
         with:
           rosdistro: ${{ matrix.env.ROS_DISTRO }}
+      - name: Get latest timestamp repos file has been edited
+        id: repos_timestamp
+        uses: vatanaksoytezer/latest-file-edit-timestamp-action@main
+        with:
+          file: moveit2.repos
       - name: Cache upstream workspace
         uses: pat-s/always-upload-cache@v2.1.5
         with:
           path: ${{ env.BASEDIR }}/upstream_ws
           key: ${{ env.CACHE_PREFIX }}-${{ github.run_id }}
           restore-keys: ${{ env.CACHE_PREFIX }}
         env:
-          CACHE_PREFIX: ${{ steps.rosdistro_release_date.outputs.date }}-upstream_ws-${{ matrix.env.IMAGE }}-${{ hashFiles('moveit2*.repos', '.github/workflows/ci.yaml') }}
+          CACHE_PREFIX: ${{ steps.rosdistro_release_date.outputs.date }}-upstream_ws-${{ steps.repos_timestamp.outputs.timestamp }}-${{ matrix.env.IMAGE }}-${{ hashFiles('moveit2*.repos', '.github/workflows/ci.yaml') }}
```

Comment:
A comment about what this does would be great.

Reply:
It's in the name of the job: https://github.com/ros-planning/moveit2/blob/vatan/fix_ci/.github/workflows/ci.yaml#L82. I don't mind adding more comment if you think is needed though.

Reply:
I updated the name to be slightly more verbose: `repos_timestamp` to `repos_edit_timestamp`

Reply:
I see. That works for me.